recipes/paraprobe/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "paraprobe" %}
+{% set version = "v0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/{{ name }}/{{ name }}-toolbox/-/archive/{{ version }}/{{ name }}-toolbox-{{ version }}.tar.gz
+    sha256: b7aba8d5d56117390ba6231c4599131019090a919ec48f84f556f49fdc41057a
+    folder: paraprobe-toolbox
+    patches:
+      - paraprobe.patch
+  - fn: v0.4.6.tar.gz
+    url: https://github.com/chr1shr/voro/archive/refs/tags/v0.4.6.tar.gz
+    folder: voro++
+    sha256: e1eb9c1f5a393604f2ca56e935b7e850644f7a3a18f5a85a5b2617f51fc70343
+
+build:
+  number: 0
+  skip: True  # [win]
+  #skip: True  # [osx]
+  #skip: True  # [py<38] 
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - git
+    - m4
+
+  host:
+    - python
+    - cython
+    - setuptools
+    - {{ mpi }}
+    - gmp
+    - mpfr
+    - libhwloc
+    - libgomp
+    - hdf5
+    - hdf5 * {{ mpi_prefix }}_*
+    - tetgen
+    - fftw
+    - boost
+    - eigen
+    - voro
+    - cgal
+    - libaec
+    - numpy >=1.17
+  run:
+    - python
+    - setuptools
+    - scikit-learn ==1.0.2
```

Comment:
A single `=` is sufficient for `conda` in contrast to `pip`.

Reply:
changed in the recipe